gdnative-core/src/core_types/vector3.rs

```diff
@@ -18,6 +18,7 @@ pub struct Vector3 {
 #[allow(clippy::unnecessary_cast)] // False positives: casts necessary for cross-platform
 #[derive(Copy, Clone, Debug, PartialEq, Eq)]
 #[repr(u32)]
+#[non_exhaustive]
 pub enum Axis {
```

Comment:
It's hard to imagine how this one is going to expand 😉

In this case it might make sense not to force users having an empty `_` branch in `match` statements.